src/pluto/pluto.py

```diff
@@ -509,6 +509,57 @@ def create_expected(df, v1, v2):
                     st.markdown(f"* in {v2} but not in {v1}:")
                     st.write(in2not1)
 
+
+    def create_outlier(df, v1, v2, condo, mapped):
+        outlier = df.loc[
+            (df.condo == condo)
+            & (df.mapped == mapped)
+            & (df.v == v1),
+            :,
+        ]
+
+        outlier_records = outlier.to_dict("records")
+        v1_outlier = [i["outlier"] for i in outlier_records if i["v"] == v1][0]
+
+        def display_dataframe(v1_outlier, field):
+            records = [i["values"] for i in v1_outlier if i["field"] == field][0]
+            if records:
+                df = pd.DataFrame(records)
+                if field == 'building_area_increase':
+                    df=df.drop(columns=['pair'])
+                df['bbl']=pd.to_numeric(df['bbl'], downcast='integer')
+                return df,df.shape[0]
```

Comment:
I think it might be simpler to just return df here, and in your else block return an empty dataframe